pkg/cliplugins/workspace/cmd/cmd.go

```diff
@@ -29,126 +29,131 @@ import (
+			return useRunE(c, args)
 		},
 	}
 
 	currentCmd := &cobra.Command{
 		Use:          "current",
-		Short:        "Returns the name of the current workspace",
+		Short:        "Replaced with \"kubectl kcp workspace\" or \"kubectl workspace\" or shorter \"kubectl ws\"",
 		Example:      "kcp workspace current",
 		SilenceUsage: true,
 		RunE: func(c *cobra.Command, args []string) error {
-			kubeconfig, err := plugin.NewKubeConfig(opts)
-			if err != nil {
-				return err
-			}
-			if err := kubeconfig.CurrentWorkspace(c.Context(), opts); err != nil {
-				return err
-			}
-			return nil
+			fmt.Println("The \"current\" command being replaced. Please do instead:\n\n  kubectl kcp workspaces\n\nor shorter:\n\n  kubectl ws")
+			return useRunE(c, nil)
 		},
 	}
 
 	listCmd := &cobra.Command{
 		Use:          "list",
-		Short:        "Returns the list of the personal workspaces of the user",
+		Short:        "Replaced with \"kubectl get workspaces\"",
 		Example:      "kcp workspace list",
 		SilenceUsage: true,
-		RunE: func(c *cobra.Command, args []string) error {
-			kubeconfig, err := plugin.NewKubeConfig(opts)
-			if err != nil {
-				return err
-			}
-			if err := kubeconfig.ListWorkspaces(c.Context(), opts); err != nil {
-				return err
-			}
-			return nil
+		Run: func(c *cobra.Command, args []string) {
+			fmt.Println("The \"list\" command is gone. Please do instead:\n\n  kubectl get workspaces\n\nIf no workspace resource is found do \"kubectl ws ..\" first to get into the parent workspace.")
 		},
 	}
 
-	useFlag := "use"
-	var typeFlag string
+	var workspaceType string
+	var enterAfterCreation bool
+	var ignoreExisting bool
 	createCmd := &cobra.Command{
 		Use:          "create",
 		Short:        "Creates a new personal workspace",
-		Example:      "kcp workspace create <workspace name>",
+		Example:      "kcp workspace create <workspace name> [--type=<type>] [--enter [--ignore-not-ready]] --ignore-existing",
 		SilenceUsage: true,
 		Args:         cobra.ExactArgs(1),
-		RunE: func(c *cobra.Command, args []string) error {
-			useAfterCreation, err := c.Flags().GetBool(useFlag)
-			if err != nil {
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Validate(); err != nil {
 				return err
 			}
+
 			kubeconfig, err := plugin.NewKubeConfig(opts)
 			if err != nil {
 				return err
 			}
-			if err := kubeconfig.CreateWorkspace(c.Context(), opts, args[0], useAfterCreation, typeFlag); err != nil {
+			if err := kubeconfig.CreateWorkspace(cmd.Context(), args[0], workspaceType, ignoreExisting, enterAfterCreation); err != nil {
 				return err
 			}
 			return nil
 		},
 	}
-	createCmd.Flags().Bool(useFlag, false, "Use the new workspace after a successful creation")
-	createCmd.Flags().StringVar(&typeFlag, "type", typeFlag, "The workspace type to create")
+	createCmd.Flags().StringVar(&workspaceType, "type", "", "A workspace type (default: Universal)")
```

Comment:
We could. This is set by defaulting in the API. I didn't want to stop us from having a default by workspace, e.g. in the root it would be Org, in the org it would be Universal.

Reply:
I see what you're saying. I hesitate any time I see manual "default: xyz" in a flag description, as it usually comes from whatever default value you specify in the flag definition.